Update service-roles-new-workspaces.md

UnkemptDataBod opened the pull request:
Clarified there are permissions that necessiate an admin enabling a feature.  Simplified the permissions table

Jak-MS commented on powerbi-docs/collaborate-share/service-roles-new-workspaces.md:
```diff
@@ -1,14 +1,14 @@
 title: Roles in workspaces in Power BI
 description: Learn about the different roles you can assign to workspace users to grant access to read, write, edit, and more.
-author: maggiesMSFT
+author: UnkemptDataBod
```
```suggestion
author: maggiesMSFT
```
authors must be Microsoft content team members

Jak-MS commented on powerbi-docs/collaborate-share/service-roles-new-workspaces.md:
```diff
@@ -62,6 +61,10 @@ Everyone in a user group gets the role that you've assigned. If someone is in se
 
 <sup>5</sup> If the items are in a workspace in a Premium capacity, you can view and interact with items in the Power BI service even if you don't have a Power BI Pro license.
 
+<sup>6</sup> If your admin has enabled this feature, see [Manage featured content](https://learn.microsoft.com/fabric/admin/service-admin-portal-featured-content)
```
```suggestion
<sup>6</sup> If your admin has enabled this feature, see [Manage featured content](/fabric/admin/service-admin-portal-featured-content)
```

Jak-MS commented on powerbi-docs/collaborate-share/service-roles-new-workspaces.md:
```diff
@@ -62,6 +61,10 @@ Everyone in a user group gets the role that you've assigned. If someone is in se
 
+<sup>6</sup> If your admin has enabled this feature, see [Manage featured content](https://learn.microsoft.com/fabric/admin/service-admin-portal-featured-content)
+
+<sup>7</sup> If your admin has enabled this feature. see [Users can try Microsoft Fabric paid features](https://learn.microsoft.com/fabric/admin/service-admin-portal-help-support#users-can-try-microsoft-fabric-paid-features)
```
```suggestion
<sup>7</sup> If your admin has enabled this feature. see [Users can try Microsoft Fabric paid features](/fabric/admin/service-admin-portal-help-support#users-can-try-microsoft-fabric-paid-features)
```

Jak-MS commented:
@maggiesMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

UnkemptDataBod commented:
@maggiesMSFT 

I have updated the PR to account for changes made since raising, there are currently no conflicts and I believe the information I have updated is relevant and correct. Has this stalled somewhere?

maggiesMSFT commented:
Hi, @UnkemptDataBod -- I've made a PR with your update, so we can view it in preview, which we can't do for updates in the public repo. So I'm closing your PR. I have sent my updated preview to the PMs to ask them to review before I sign off. Thanks!
Replacing with https://github.com/MicrosoftDocs/powerbi-docs-pr/pull/11238
